config/_default/menus/menus.en.yaml

```diff
@@ -323,10 +323,15 @@ main:
     url: agent/basic_agent_usage/windows/
     weight: 117
     parent: basic_agent_usage
+  - name: SCCM
```

Comment:
@hestonhoffman I do understand this fits under Windows, but I'm wondering if this should be listed in alphabetical order like the other platforms?